### PR TITLE
Added sections on the battle sprite properties

### PR DESCRIPTION
**Pull request by turtleisaac**
# Battle Sprite Properties

# Description

Added new files and resources (gifs) pertaining to the battle sprite property files (height.narc, pl_poke_data.narc, and their HGSS equivalents) talking about the structure of the files and showing examples of the movement types

# Which Generation (4/5/Universal)?

Gen 4, and is universal across all of gen 4

# Notes

TODO
* need to work on getting clips for examples of every back sprite movement
* there's tons of undocumented bytes in the poke_data structure that need research

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
         - [Code Injection](gen5/universal/guides/code_injection/code_injection.md)  
 - [Universal]()
     - [Code Injection](universal/guides/code_injection/code_injection.md)
-    - [Sprite Indexing](universal/guides/sprite_indexing/indexing.md)
+    - [Sprite Indexing<sup>*(WIP)*</sup>](universal/guides/sprite_indexing/indexing.md)
```
Comment (BluRosie) [inline src/SUMMARY.md:22]: it's not still wip, is it?

File: src/SUMMARY.md [hunk 2]
```diff
@@ -29,6 +29,15 @@
...
             - [Town Map Structures](gen4/hgss/structures/town_map/town_map_spots.md)
+    - [Universal]()
+      - [Structures]()
+        - [Battle Sprite Properties](gen4/universal/structures/battle_sprites/battle_sprite_properties.md)
+          - [Sprite Poke_Data NARC Structure<sup>*(WIP)*</sup>](gen4/universal/structures/battle_sprites/poke_data.md)
+          - [Sprite Height NARC Structure](gen4/universal/structures/battle_sprites/height.md)
+      - [Resources]()
+        - [Battle Sprite Properties](gen4/universal/structures/battle_sprites/battle_sprite_properties.md)
+          - [Front Sprite Movement Type Examples<sup>*(WIP)*</sup>](gen4/universal/structures/battle_sprites/front_movement_types.md)
+          - [Back Sprite Movement Type Examples<sup>*(WIP)*</sup>](gen4/universal/structures/battle_sprites/back_movement_types.md)
```
Comment (BluRosie) [inline src/SUMMARY.md:40]: do we want to add universal section to each gen?  this is relevant for #19 as well

File: src/gen4/universal/structures/battle_sprites/poke_data.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+### Sections
+* [Movement Types](#movement-types)
+* [ShadowSize](#shadowsize)
+
+### Movement Types
+
+> WIP: need to figure out the gigantic mess that is literally everything to do with the AnimationSubTable entries
+* See [Front sprite movement types](front_movement_types.md)
+  * *Note*: With the discovery of the function of further elements in the struct, this should no longer to be considered an accurate reflection of what each value does. As we understand the AnimationSubTable data further, attempts will be made to provide a better system for displaying examples of all possible animations. However, in the meantime, this can still give you a rough idea of what each movementID value looks like.
```
Comment (BluRosie) [inline src/gen4/universal/structures/battle_sprites/poke_data.md:122]: add this note to `front_movement_types.md` as well

**Review by BluRosie — CHANGES_REQUESTED** [review]
just a few things that i left comments on

File: src/SUMMARY.md (hunk 1 above)
Comment (turtleisaac) [reply src/SUMMARY.md:22]: WIP in the sense that there’s only a guide for GIMP currently

**Pull request closed without merging**